clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -932,6 +932,23 @@ raw2trace_t::read_header(raw2trace_thread_data_t *tdata,
         header->cache_line_size = proc_get_cache_line_size();
         unread_last_entry(tdata);
     }
+
+    in_entry = get_next_entry(tdata);
```

Comment:
See comment above: to handle prctl this will have to be in the general mid-trace processing, so should we start out with it there instead of having to look for this marker in two different places?

Reply:
Done.